test/onnx/onnx_test_common.py

```diff
@@ -63,6 +64,12 @@
 pytorch_operator_dir = os.path.join(onnx_model_dir, "pytorch-operator")
 
 
+class TorchModelType(Enum):
+    UNDEFINED = 0
```

Comment:
briefly explain in comments what UNDEFINED is for?

Reply:
It can be used for optional argument with no default values assigned
e.g. `def my_func(model_type: TorchModelType = TorchModelType.UNDEFINED)`

Reply:
should we make it required argument though? I guess the question is, is there a case that we don't raise if model type is UNDEFINED?

Reply:
Makes sense; I'll remove `UNDEFINED`